modules/apps/27-interchain-accounts/controller/types/msgs.go

```diff
@@ -75,10 +72,6 @@ func (msg MsgSendTx) ValidateBasic() error {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "failed to parse owner address: %s", msg.Owner)
 	}
 
-	if msg.TimeoutHeight.IsZero() && msg.TimeoutTimestamp == 0 {
```

Comment:
This should return an error if the timestamp is 0, otherwise it is expected we are sending a packet which should automatically timeout